job/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -419,7 +419,14 @@ private String key() {
             System.err.printf("loading %s @%h%n", this, this);
         }
         if (execution != null) {
-            execution.onLoad();
+            try {
+                execution.onLoad(new Owner(this));
+            } catch (IOException x) {
+                LOGGER.log(Level.WARNING, null, x);
```

Comment:
Not sure what useful text to put there. An exception was caught; I have no real prediction about what it might be about. If it ever happens, the stack trace is going to be the only interesting thing.